app.js

```diff
@@ -1,3 +1,5 @@
+// For review
+
 // Array of markers
 let markers = [
```

Comment:
If markers will never be rewritten, this is good place to use `const`!

Reply:
Thanks for the reminder, this has been updated!